sample-app/app/controllers/users_controller.rb

```diff
@@ -1,3 +1,24 @@
 class UsersController < ApplicationController
-  def new; end
+  def new 
+    @user = User.new
+  end
+
+  def show 
+    @user = User.find(params[:id])
+  end
+
+  def create
+    @user = User.new(user_params)
```

Comment:
chỗ nào bỏ dc ngoặc thì bỏ nhé e
tự kiểm tra toàn bộ và bỏ các ngoặc k cần thiết đi nhé